src/dtos/confirmed-import-dto.ts

```diff
@@ -1,13 +0,0 @@
-import { ViewError } from './view-error';
-import { FileImportDTO } from './dataset-dto';
-
-export interface ConfirmedImportDTO {
-    success: boolean;
-    fileImport: FileImportDTO;
-}
-
-export interface UploadErrDTO {
-    success: boolean;
-    fileImport: FileImportDTO | undefined;
-    errors: ViewError[];
-}
```

Comment:
These should no-longer be required as the api service will throw exceptions instead.